test/pom.xml

```diff
@@ -159,7 +159,7 @@
 						<webResource>
 							<directory>src/main/webapp/META-INF</directory>
 							<targetPath>META-INF</targetPath>
-							<filtering>true</filtering>
+							<filtering>${filteringMetaInf}</filtering>
```

Comment:
Is this filtering used somewhere? If not, we can remove it.

Reply:
I did not remove it, I made it configurable with true as default. Or are you talking about filteringDeploymentDescriptors? See my earlier comment on that:

I think element filteringDeploymentDescriptors can be removed (no ${*} in web.xml and couldn't find a reason in the commit that added it) but in case it is needed for something I can also add a property filteringDeploymentDescriptors.

Reply:
Wat gebeurd er als je dit hele `<filtering>` blok weglaat? Volgens mij gebruiken we het nergens!
Dan hoeft het ook niet optioneel te worden ;)

Reply:
Wordt dat niet gebruikt om alle ${*} (zoals ${databaseHost}) in de context.xml te vervangen? Wat mij betreft kan het weg maar zal er toch niet voor niks in gezet zijn? Als we filtering weghalen wat doen we dan met alle ${*} in context.xml? Of is dat meer een soort documentatie dat het vervangen moet worden met een andere waarde als je er echt iets mee wil doen?

Reply:
De `${databaseHost}` mag vervangen worden door `host.docker.internal` (dns naam). De overige properties moet tomcat zelf resolven, dat moet niet uit het project/maven komen. (zoals de tibco connectie info)

Martijn had ooit het filtering bedacht om er voor te zorgen dat hij dynamisch het endpoint aan kon passen. Maar daar gebruiken we nu docker voor. Ik denk dat we het nooit goed om hebben gezet ...

Reply:
Ok, nice. Ik heb het verwijderd.